ui-admin/src/study/participants/datarepo/DatasetList.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react'
+import { getDatasetDashboardPath, StudyEnvContextT } from 'study/StudyEnvironmentRouter'
+import Api, { DatasetDetails } from 'api/api'
+import LoadingSpinner from 'util/LoadingSpinner'
+import { Store } from 'react-notifications-component'
+import { failureNotification } from 'util/notifications'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faExternalLink } from '@fortawesome/free-solid-svg-icons/faExternalLink'
+import { instantToDefaultString } from '../../../util/timeUtils'
+import {
+  ColumnDef,
+  getCoreRowModel,
+  getSortedRowModel,
+  SortingState,
+  useReactTable
+} from '@tanstack/react-table'
+import { basicTableLayout } from '../../../util/tableUtils'
+import { Link } from 'react-router-dom'
+import { faPlus } from '@fortawesome/free-solid-svg-icons/faPlus'
+import { useUser } from '../../../user/UserProvider'
+import CreateDatasetModal from './CreateDatasetModal'
+
+const datasetColumns = (currentEnvPath: string): ColumnDef<DatasetDetails>[] => [{
+  id: 'datasetName',
+  header: 'Dataset Name',
+  accessorKey: 'datasetName',
+  cell: info => <Link to={getDatasetDashboardPath(info.getValue() as unknown as string, currentEnvPath)}
+    className="mx-2">{info.getValue() as unknown as string}</Link>
+}, {
+  id: 'datasetUuid',
+  header: 'Dataset ID',
+  accessorKey: 'datasetId'
+}, {
+  id: 'created',
+  header: 'Created Date',
+  accessorKey: 'createdAt',
+  cell: info => instantToDefaultString(info.getValue() as unknown as number)
+}, {
+  id: 'lastUpdated',
+  header: 'Last Export',
+  accessorKey: 'lastExported',
+  cell: info => instantToDefaultString(info.getValue() as unknown as number)
+}, {
+  header: 'Terra Data Repo',
+  accessorKey: 'datasetId',
+  cell: info => <a href={
+      `https://jade.datarepo-dev.broadinstitute.org/datasets/${info.getValue()}`} target="_blank"
+  >View in Terra Data Repo <FontAwesomeIcon icon={faExternalLink}/></a>
+}]
+
+const DatasetList = ({ studyEnvContext }: {studyEnvContext: StudyEnvContextT}) => {
+  const { currentEnvPath } = studyEnvContext
+  const [showCreateDatasetModal, setShowCreateDatasetModal] = useState(false)
+  const [datasets, setDatasets] = useState<DatasetDetails[]>([])
+  const [isLoading, setIsLoading] = useState(true)
+  const [datasetsSorting, setDatasetsSorting] = React.useState<SortingState>([])
+  const { user } = useUser()
+
+  const datasetTable = useReactTable({
+    data: datasets,
+    columns: datasetColumns(currentEnvPath),
+    state: {
+      sorting: datasetsSorting
+    },
+    onSortingChange: setDatasetsSorting,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel()
+  })
+
+  const contentHeaderStyle = {
+    padding: '1em 0 0 1em',
+    borderBottom: '1px solid #f6f6f6'
+  }
+
+  const loadData = async () => {
```

Comment:
Same here, as in `DatasetDashboard` but without the `datasetName` consideration.